dts/bindings/opamp/st,stm32-opamp.yaml

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) 2025 Alexander Kozhinov <ak.alexander.kozhinov@gmail.com>
+# SPDX-License-Identifier: Apache-2.0
+
+description: STM32 OPAMP (Operational Amplifier)
+
+include:
+  - opamp-controller.yaml
+
+compatible: st,stm32-opamp
+
+properties:
+  reg:
+    required: true
+  clocks:
+    required: true
+  pinctrl-0:
+    required: true
```

Comment:
Add also `clock-names` and `pinctrl-names` as required.

Reply:
added